api/users/serializers.py

```diff
@@ -116,7 +90,10 @@ def update(self, instance, validated_data):
         for attr, value in validated_data.items():
             if 'social' == attr:
                 for key, val in value.items():
-                    instance.social[key] = val
+                    if key == 'profileWebsites':
+                        instance.social[key] = val
+                    else:
+                        instance.social[key] = val[0]
```

Comment:
So in the case of profileWebsites, we'll replace the list with what's passed in here, but for everything else, we take the first item from the list and discard the rest silently? I think this needs to be documented in the view as to which items are currently accepting multiple items, and if there are more than one items passed in to a social field that is currently only accepting one item, that an error should be raised telling them that those fields are not currently accepting more than one item.